packages/wdio-browserstack-service/src/service.ts

```diff
@@ -95,7 +97,12 @@ export default class BrowserstackService implements Services.ServiceInstance {
             }
         } else {
             // Mocha
-            this._fullTitle = `${test.parent} - ${test.title}`
+            // eslint-disable-next-line no-lonely-if
```

Comment:
This rule is quite valid and I don't agree disabling it: https://eslint.org/docs/latest/rules/no-lonely-if